content/200-concepts/100-components/02-prisma-client/090-raw-database-access.mdx

```diff
@@ -157,6 +157,53 @@ prisma.$queryRawUnsafe(
 $queryRawUnsafe<T = unknown>(query: string, ...values: any[]): PrismaPromise<T>;
 ```
 
+### Parameterized queries
+
+As an alternative to tagged templates, `$queryRaw` and `$executeRaw` support standard parameterized queries where each variable is represented by a symbol (`?` for mySQL, `$1`, `$2`, and so on for PostgreSQL). The following example uses a MySQL query:
```

Comment:
What about SQLite? What about SQL Server? Since it goes GA we should make it clear too

Reply:
Lets create an issue and take it after. Im currently chatting with @millsp about this page and adding those examples might take time

Reply:
I'll see what he says

Reply:
I guess that I would only list SQLite and SQL Server next to the approach both use (I guess `?`) and call it a day without creating new examples, but it's also fine to postpone since it was like that before (it seems)

Reply:
https://github.com/prisma/docs/issues/2288